src/components/Auth/components/Login/Login.tsx

```diff
@@ -1,13 +1,16 @@
 import { useFormik } from "formik";
+import { useRouter } from "next/router";
 import * as React from "react";
+import { useState } from "react";
 import * as Yup from "yup";
 import { Typography } from "@mui/material";
+import { supabase } from "@src/utils/supabaseClient";
 import * as Styles from "./styles";
 
 const ValidateFormSchema = Yup.object().shape({
-  username: Yup.string()
+  email: Yup.string()
```

Comment:
Yup tiene el método email con el que puedes validar que tenga la estructura de un email
```suggestion
  email: Yup.string().email()
```